e2e/various-suite/frontend-sandbox-datasource.spec.ts

```diff
@@ -85,6 +85,8 @@ describe('Datasource sandbox', () => {
         e2e.pages.Explore.visit();
         e2e.components.DataSourcePicker.container().should('be.visible').click();
         cy.contains(DATASOURCE_TYPED_NAME).scrollIntoView().should('be.visible').click();
+        // make sure the datasource was correctly selected and rendered
+        cy.get('span[data-testid="data-testid SandboxDatasourceInstance breadcrumb"]').should('exist');
```

Comment:
should be able to do `e2e.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME).should('be.visible');` instead everywhere

Reply:
Yes I just changed it